cypress/integration/e2e/homepage_spec.js

```diff
@@ -0,0 +1,68 @@
+describe('Home page tests', function()
+{
+    const newProjectName = 'Toms Team';
+
+    beforeEach(function() {
+        cy.visit('');
+        // Initialise Fixture objects
+        cy.fixture('elements.json').as('element');
+    })
+
+    it('Test to ensure the homepage displays the App Logo', function()
+    {
+        cy.get(this.element.appHeaderElements.appLogo)
```

Comment:
If you add an index file into your `fixtures` directory (although I would do this in a `selectors` directory, you should be able to import these at the top of this file, for example.

`index.js` would include:

```
import elements from './elements.json';

export default {
  elements
}
```
NOTE: I would break the contents of `elements.json` into their own files too (similar to how you would in selenium).

You could then at the top of this file do:
`import { elements } from '../../fixtures';`

And use it in your tests as `element.appHeaderElements.appLogo`.

As you can imagine, breaking the selectors up into separate files and placing them in a selectors directory, would mean you could implement slight changes which in my opinion look nicer.

You would have created a `selectors` directory with `appHeader.json` in. You could then use it in this file as:
'import { appHeader } from '../../selectors';`

Then used as `appHeader.appLogo`.

Looks much cleaner and breaks the files up nicely :D